packages/dashboard/.storybook/main.js

```diff
@@ -26,6 +28,24 @@ module.exports = {
             buffer: 'buffer',
             path: 'path-browserify',
         }
+        config.resolve.alias = {
```

Comment:
Do you encountered some problem so you need to do this?

Reply:
Yes, try to fix CI. Do you have any suggestion?

Reply:
ok it's because you import `mask` package from the `dashboard` package. that should not happen